packages/ui5-application-writer/src/index.ts

```diff
@@ -133,3 +134,4 @@ async function enableTypescript(basePath: string, fs?: Editor): Promise<Editor>
 
 export { Ui5App, generate, enableTypescript, isTypescriptEnabled };
 export { App, Package, UI5, AppOptions };
+export { generateReadMe, ReadMe };
```

Comment:
Is this necessary, I thought readme should be now generated by default or if not should be an option. Does it need to be called separatly?

Reply:
read me has now been moved to fiori generator shared